### PR TITLE
More core sdk examples- user, balance, transfers, withdrawals

### PR DESCRIPTION
**Pull request by cdrabner95**
# Summary

Adds examples to the core-sdk for functionality with withdrawals, transfers, balances, users, and projects.

# Why the changes

Support team's project to flesh out all core-sdk code examples.

# Before merging
- [ ] ***For Immutable developers:*** Do the changes in this PR require documentation updates? Please refer to this [SDK documentation guide](https://immutable.atlassian.net/wiki/spaces/PPS/pages/1916994017/SDK+documentation+guide) and list links to documentation update PRs (they don't have to be merged) below (or write `N/A`):
NA

### CODE REVIEW

**Review by sprngbk — CHANGES_REQUESTED** [review]

File: examples/README.md [hunk 1]
```diff
@@ -212,4 +212,90 @@ Update [getToken.ts](./getToken.ts#L10) with the token contract address then run
 
 ```sh
 yarn get-token
+```
+
+## Get Balance
```
[the remark this answers is not in the document]
Comment (cdrabner95) [reply examples/README.md]: fixed

File: examples/completeErc20Withdrawal.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+(async () => {
+  try {
+    const walletConnection = await generateWalletConnection('goerli');
+
+    const client = new ImmutableX(Config.SANDBOX);
+
+    const starkPublicKey = await walletConnection.starkSigner.getAddress();
+
+
```
[the remark this answers is not in the document]
Comment (cdrabner95) [reply examples/completeErc20Withdrawal.ts]: done

File: examples/getProjects.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      console.log('projects', projects);
+    } catch (err) {
+      console.error(err);
+      process.exit(1);
+    }
+  })();
+
+
+
+
```
[the remark this answers is not in the document]
Comment (cdrabner95) [reply examples/getProjects.ts]: done

File: examples/listWithdrawals.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        tokenType:"", // ERC20, ERC721, Eth
+        cursor:"", // cursor
+        direction: "", //asc or desc
+        user: "", // eth address of user who submittedd withdrawal request
+        status: "", // success / failure
+        tokenAddress: "", // contract address of token
+        tokenId: "", // token ID of NFT if applicable 
+        orderBy: "" // transaction_id, updated_at, created_at
+
+
```
Comment (CodeSchwert) [inline examples/listWithdrawals.ts:20]: Extra lines here

**Review by CodeSchwert — APPROVED** [review]

**Review by Nathan-Ballantyne — APPROVED** [review]

**Pull request merged**